.github/dependabot.yml

```diff
@@ -42,7 +42,7 @@ updates:
       - /docker/base
       - /docker/debug
       - /examples/hotrod
-      - /plugin/storage/cassandra
+      - /internal/storage/cassandra
```

Comment:
I prefer `internal/storage/v1/...` and then v2 accordingly.

Reply:
@yurishkuro but aren't we going to do the upgrades in place?

Reply:
in-place can still be done with versioned packages. For example:
  * move cassandra to v2, leave only aliases / redirect in v1
  * update v2 in place to support v2 API, change aliases/redirect in v1 to adapter
  * update callers to instantiate v2 directly

I think having a clear separation would be useful and more intuitive.

Let's document proposed directory structure in the ticket first (similar to what I had in a chat but nesting everything under /internal/).

Reply:
@yurishkuro Updated the ticket - can you let me know if it looks good?

Reply:
lgtm